code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -107,15 +107,21 @@
 	if(cooldowntime > world.time)
 		to_chat(user, "<span class='cultitalic'>The magic in [src] is weak, it will be ready to use again in [getETA()].</span>")
 		return
-	var/choice = alert(user,"You study the schematics etched into the forge...",,"Shielded Robe","Flagellant's Robe","Nar-Sien Hardsuit")
+	var/choice = alert(user,"You study the schematics etched into the forge...",,"Shielded Robe","Flagellant's Robe","Bastard Sword")
 	var/pickedtype
 	switch(choice)
 		if("Shielded Robe")
 			pickedtype = /obj/item/clothing/suit/hooded/cultrobes/cult_shield
 		if("Flagellant's Robe")
 			pickedtype = /obj/item/clothing/suit/hooded/cultrobes/berserker
-		if("Nar-Sien Hardsuit")
-			pickedtype = /obj/item/clothing/suit/space/hardsuit/cult
+		if("Bastard Sword")
+			if(world.time < 18000)
```

Comment:
this means you can only create the weapon in the first 30 minutes, I assume you want the opposite (and your op also says 20, so, you probably meant 12000)

Reply:
Thanks for catching that, I think I had initially planned to have the first statement be the rejection message, or maybe I left it that way for testing, either way I need to change it.

Reply:
Use an offset of the SSticker var for game start time instead of 0